src/gtest/test_bip32.cpp

```diff
@@ -1,32 +1,38 @@
 // Copyright (c) 2013 The Bitcoin Core developers
+// Copyright (c) 2021 The Pastel developers
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or https://www.opensource.org/licenses/mit-license.php .
 
-#include <boost/test/unit_test.hpp>
 
 #include "key.h"
 #include "key_io.h"
 #include "uint256.h"
 #include "util.h"
 #include "utilstrencodings.h"
-#include "test/test_bitcoin.h"
+#include "vector_types.h"
 
+#include <gtest/gtest.h>
```

Comment:
We can use the following logic when adding includes:
make 3 separate groups of includes:
    - standard c++ includes
    - 3rdparty includes, like gtest.h. json.hpp, etc... all depends should go here
    - src includes:
         1.  more common includes (utils, encoding, etc) should go first, 
         2. then Pastel includes like src/pastelid, src/mnode/...
         3. any test-specific includes